src/interfaces/store-interfaces/state-interfaces/correction-information-interface.ts

```diff
@@ -24,11 +25,7 @@ export interface CorrectionInformationIF {
     naicsCode: string
     naicsDescription: string
   }
-  contactPoint?: {
-    email: string
-    phone: string
-    extension?: number
-  }
+  contactPoint?: ContactPointIF
```

Comment:
We need interface `ContactPointIF` otherwise [`filing-template-mixin.ts`](https://github.com/bcgov/business-edit-ui/blob/2f672d328d1ed0b5db60d714912a7cf67ac25bd5/src/mixins/filing-template-mixin.ts#L166) will have an error.